src/main/java/test/A.java

```diff
@@ -0,0 +1,9 @@
+package test;
+
+public class A {
+  Object o;
+  public static void main(String...args) {
+    A a = new A();
+    a.o.toString();
```

Comment:
Potential **null** derefence

Reply:
Do you mean from that line?
https://github.com/johspaeth/PathExpression/blob/199af867c7687a328b1307e19485a3914188cc0d/src/main/java/test/A.java#L4

Reply:
https://github.com/johspaeth/PathExpression/blob/199af867c7687a328b1307e19485a3914188cc0d/src/main/java/test/A.java#L5